dspace/solr/search/conf/schema.xml

```diff
@@ -181,7 +183,8 @@
             <tokenizer class="solr.KeywordTokenizerFactory"/>
 
             <!--Lower cases our values-->
-            <filter class="solr.LowerCaseFilterFactory" />
+            <!-- do not do this as we are using typeahead and we need real values
```

Comment:
why

Reply:
I copied set up from the v5 (it is OK's update)